tests/baselines/reference/unionTypeCallSignatures.js

```diff
@@ -27,9 +27,9 @@ unionOfDifferentNumberOfSignatures(10); // error - no call signatures
 unionOfDifferentNumberOfSignatures("hello"); // error - no call signatures
 
 var unionWithDifferentParameterCount: { (a: string): string; } | { (a: string, b: number): number; } ;
-unionWithDifferentParameterCount();// no  call signature
-unionWithDifferentParameterCount("hello");// no  call signature
-unionWithDifferentParameterCount("hello", 10);// no  call signature
+unionWithDifferentParameterCount();  //error TS2346
```

Comment:
Can you not hard-code the error message codes? This makes it harder to identify the problem.